app/views/admin_merchants/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2>Create New Merchant</h2>
+<br>
+<%= form_with url:"/admin/merchants", method: :post, data: {turbo: false} do |form| %>
+  <%= form.label :name %><br>
+  <%= form.text_field :name %><br>
+  <%= form.submit "Submit" %>
+<% end %>
```

Comment:
when we refactor we could look at moving this form into using model: @merchant instead of the form_with